datasource-sdk/src/main/java/org/apache/kylin/sdk/datasource/adaptor/PostgresqlAdaptor.java

```diff
@@ -34,7 +35,12 @@ public PostgresqlAdaptor(AdaptorConfig config) throws Exception {
     public List<String> listDatabases() throws SQLException {
         List<String> ret = new ArrayList<>();
         try (Connection con = getConnection()) {
-            ret.add(con.getCatalog());
+            DatabaseMetaData metaData = con.getMetaData();
```

Comment:
Should the implementation here be consistent with `PostgresqlJdbcMetadata#listDatabases `?